rosdep/python.yaml

```diff
@@ -6173,6 +6173,12 @@ python3-numpy:
   openembedded: [python3-numpy@openembedded-core]
   rhel: ['python%{python3_pkgversion}-numpy']
   ubuntu: [python3-numpy]
+python3-numpy-stl:
+  debian: [python3-numpy-stl]
+  fedora: [python3-numpy-stl]
+  ubuntu: 
+    '*': [python3-numpy-stl]
+    xenial: [null]
```

Comment:
should be

```suggestion
    xenial: null
```

?

Reply:
I think checkers are failing because of that

Reply:
@ivanpauno thanks for that!